HW_6/task_3.py

```diff
@@ -0,0 +1,13 @@
+def is_prime(num):
+    """Check if the number is prime or not
+    """
+    if num == 1:
+        return "The number should be grater than 1!"
+    counter = 0
+    for i in range(2, num // 2 + 1):
+        if num % i == 0:
+            counter += 1
+    if counter <= 0:
+        return True
+    else:
+        return False
```

Comment:
def is_prime(num):
    """Check if the number is prime or not
    """
    if num == 1:
        return "The number should be grater than 1!"
    counter = 0
    for i in range(2, num // 2 + 1):
        if num % i == 0:
            counter += 1
            return False
    return True


can be modified like this. So you do not need additional if/else